deconstrst/deconstrst.py

```diff
@@ -29,6 +34,26 @@ def build(srcdir, destdir):
 
     return app.statuscode
 
+def get_conf_builder():
+    conf_file = open('conf.py')
+
+    try:
+        conf_data = conf_file.read()
+    finally:
+        conf_file.close()
+
+    try:
+        code = compile(conf_data, 'conf.py', 'exec')
+    except SyntaxError:
+        raise
+
+    exec(code)
+    local_vars = locals()
+
+    try:
+        return local_vars['builder']
+    except KeyError:
+        return DEFAULT_BUILDER
```

Comment:
I didn't realize it was so awkward to pull this from `conf.py`, by the way! I thought you'd be able to pull it the same way that I'm pulling the other `conf.py` variable, but of course you don't have a Builder here, yet.

Would it be easier to put this in `_deconst.json` instead? Although, that makes `_deconst.json` preparer-specific... That would probably be okay, considering that you can't really swap out preparers easily.